06_building_stuff/src/utils/hooks/useRestroMenu.js

```diff
@@ -22,9 +22,12 @@ export default function useRestroMenu(resId) {
 
   const setRestroData = (menuInfo) => {
     setResName(menuInfo?.data?.cards[2]?.card?.card?.info.name);
+    // Finding the first card within the `cards` array using find method that 
+    // has an itemCards property and then only access the rest of its cards
     setResInfo(
-      menuInfo?.data?.cards[4]?.groupedCard?.cardGroupMap?.REGULAR?.cards[3]
-        ?.card?.card?.itemCards
+      menuInfo?.data?.cards[4]?.groupedCard?.cardGroupMap?.REGULAR?.cards.find(
```

Comment:
Instead of statically accessing a single array element, I am using `.find` method to find specific array element which has the `itemCards` object only then I am accessing the rest of its nested objects.

// Finding the first card within the `cards` array using find method that 
// has an itemCards property and then only access the rest of its cards

Reply:
This way any array element which has that card will dynamically be found based on `itemCards` element